core-bundle/src/Image/ImageSizes.php

```diff
@@ -107,17 +107,32 @@ private function loadOptions(): void
         $this->options = $GLOBALS['TL_CROP'];
 
         $rows = $this->connection->fetchAll(
-            'SELECT id, name, width, height FROM tl_image_size ORDER BY pid, name'
+            'SELECT
+                s.id, s.name, s.width, s.height, t.name as theme
+            FROM
+                tl_image_size s
+            LEFT JOIN
+                tl_theme t ON s.pid=t.id
+            ORDER BY
+                s.pid, s.name'
         );
 
+        $options = [];
```

Comment:
I think we should not use an extra array here, but work on `$this->options` directly. The current implementation overwrites groups from `$GLOBALS['TL_CROP']` instead of extending it (can be testet by naming the theme “relative” or “exact”).

Removing line 120 and 135 and replacing `$options` with `$this->options` fixed the issue.

Reply:
I see. The idea behind it was to show the predefined image sizes on top. When working on `$this->options` directly, they are added at the end of the array.

Reply:
Also, I'm not sure if naming a theme "relative" or "exact" is a supported use case. 😄

Reply:
To get the order right we could use:

```php
$this->options = array_merge_recursive($options, $this->options);
```

I agree that it might not be a supported use-case, but it should at least not remove other image sizes I think.

Reply:
Fixed in 444d5d3611f9648796cfb1a8e9577bbc21acbef8